diff --git a/packaging/macos/build.sh b/packaging/macos/build.sh
index 58d8159..8490471 100755
--- a/packaging/macos/build.sh
+++ b/packaging/macos/build.sh
@@ -370,7 +370,7 @@ function build_qt {
        --parallel $DRAKETHREADS
 
   # cd ${CMPL}/${package}
-  build_tarball command "make DESTDIR=TMPDIR install"
+  build_tarball command "cmake --install . --prefix TMPDIR${TARGET}"
 
   CXXFLAGS=$saved_CXXFLAGS
 }
