diff --git a/packaging/macos/build.sh b/packaging/macos/build.sh
index 58d8159..7d8ef15 100755
--- a/packaging/macos/build.sh
+++ b/packaging/macos/build.sh
@@ -511,6 +511,9 @@ EOF
 
   ${SCRIPT_PATH}/fix_library_paths.sh ${dmgmac}/**/*.dylib(.) ${dmgmac}/{mkvmerge,mkvinfo,mkvextract,mkvpropedit,mkvtoolnix-gui}
 
+  # Strip debug symbols from all dylibs and plugins
+  strip -x ${dmgmac}/**/*.dylib(.)
+
   if [[ -n ${SIGNATURE_IDENTITY} ]]; then
     typeset -a non_executables
     for FILE (${dmgcnt}/**/*(.)) {
