--- a/qtbase/src/corelib/thread/qyieldcpu.h
+++ b/qtbase/src/corelib/thread/qyieldcpu.h
@@ -9,6 +9,10 @@
 #include <QtCore/qprocessordetection.h>
 #include <QtCore/qtconfigmacros.h>

+#if defined(Q_PROCESSOR_ARM) && __has_include(<arm_acle.h>)
+#  include <arm_acle.h>
+#endif
+
 #ifdef Q_CC_MSVC_ONLY
 // MSVC defines _YIELD_PROCESSOR() in <xatomic.h>, but as that is a private
 // header, we include the public ones
